internal/backend/sessions/sessioncalls/calls.go

```diff
@@ -58,6 +55,11 @@ type calls struct {
 	//                 user script.
 	generalWorker worker.Worker
 	uniqueWorker  worker.Worker
+
+	// Pass executors for session specific calls (functions that are
+	// defined in runtime script, as opposed to integrations).
+	executorsForSessions   map[sdktypes.SessionID]*sdkexecutor.Executors
+	executorsForSessionsMu sync.RWMutex
```

Comment:
The usual practice is to place the lock variable on top of the variables it guards.

Reply:
news to me, but ok